salt/metalk8s/archives/unmounted.sls

```diff
@@ -0,0 +1,10 @@
+{% set removable_archives = salt.metalk8s.get_removable_archives() %}
+{% for env, archive in removable_archives.items() %}
+{% if archive.iso %}
+Archive {{ archive.iso }} absent at {{ archive.path }}:
+  mount.unmounted:
+    - name: {{ archive.path }}
+    - device: {{ archive.iso }}
+    - persist: true
```

Comment:
It properly clean the fstab entry ? Cool :+1:

Reply:
Yep, just checked (also looked at `/etc/fstab`, to be sure :innocent:):

```
          ID: Unmount '/srv/scality/releases/metalk8s-2.11.7.iso' from '/srv/scality/metalk8s-2.11.7'
    Function: mount.unmounted
        Name: /srv/scality/metalk8s-2.11.7
      Result: True
     Comment: Target was successfully unmounted. Removed target from fstab
     Started: 10:09:59.285132
    Duration: 262.827 ms
     Changes:   
              ----------
              persist:
                  purged
              umount:
                  True
```